src/frontend/js/contexts/SessionContext/JoanieSessionProvider.tsx

```diff
@@ -113,11 +116,13 @@ const JoanieSessionProvider = ({ children }: React.PropsWithChildren<any>) => {
     () => ({
       user,
       isLoadingUser,
+      joanieUser: joanieUser.item,
+      isLoadingJoanieUser: joanieUser.states.fetching,
```

Comment:
Why not just exposing one fetching indicator ?

Reply:
I follow what's exist for User:
```ts
user,
isLoadingUser,
```